Makefile

```diff
@@ -0,0 +1,32 @@
+SHELL := bash
+
+PYTHON := $(shell command -v python || command -v python3)
+$(if $(PYTHON),,$(error No python or python3 binary found))
+
+export PYTHONPATH := $(PWD)
+
+TESTS := \
+    test_interpreter \
+    test_vm_compiler \
+    test_barevm \
+
+
+default:
+
+test: $(TESTS)
+
+test_barevm:: barevm/barevm
```

Comment:
You could also run barevm/barevm_unittest while you're at it

Reply:
Sounds good. Just a minute!

Reply:
fixed in 313f349

Reply:
PS Please make sure I got the test invocation right. Thanks.